src/python/benchmark_models_petab/base.py

```diff
@@ -1,12 +1,26 @@
 """Get a petab problem from the collection."""
 
-import os
+from pathlib import Path
 
 import petab
 
 from .C import MODELS_DIR
 
 
+def get_problem_yaml_path(id_: str) -> Path:
+    """Get the path to the PEtab problem YAML file.
+
+    Parameters
+    ----------
+    id_: Problem name, as in `benchmark_models_petab.MODELS`.
+
+    Returns
+    -------
+    The path to the PEtab problem YAML file.
+    """
+    return Path(MODELS_DIR, id_, id_ + ".yaml")
```

Comment:
In case we go for allowing benchmark models produced with `petab.Problem.to_files_generic`, which is my preference since the ID in file names is extraneous given the directory name. I would already try that for this Bertozzi update: https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/pull/222

```suggestion
    yaml_path = Path(MODELS_DIR, id_, id_ + ".yaml")
    if not yaml_path.exists():
        yaml_path = Path(MODELS_DIR, id_, "problem.yaml") 
    if not yaml_path.exists():
        raise ValueError(f"Could not find YAML for problem with ID `{id_}`.")
    return yaml_path
```

Reply:
Either works for me.